ReferenceManual.py

```diff
@@ -0,0 +1,10 @@
+import CableColorPairs
+
+def referenceManualCreation():
+    number_start = 1
+    number_end = 27     
```

Comment:
The number 27 is not intuitive. A maintainer could make a mistake when a color is added. How can we make it more intuitive?